### PR TITLE
Partial Filipino Translation - 1 177 translated words

### PR DESCRIPTION
**Pull request by mind-blast**
Partial Translating File request-response.rst with 1 177 translated words.
Note : Code text and Computer Generated Code Text are not counted as translated words.

### CODE REVIEW

File: tl/controllers/request-response.rst [hunk 1]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
+---------------
 
-To access the session for a given request use the ``session()`` method::
+Para ma-access ang sesyon para sa ibinigay na hiling na ginamit sa ``session()`` na pamamaraan::
 
     $userName = $this->request->session()->read('Auth.User.name');
 
-For more information, see the :doc:`/development/sessions` documentation for how
-to use the session object.
+Para sa karagdagangn impormasyon, tingnan ang :doc:`/development/sessions` na dokumentasyon para sa kung papaano
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `karagdagang`

File: tl/controllers/request-response.rst [hunk 2]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
-Returns the host your application is on::
+Binabalik sa host ang iyong aplikasyon sa::
 
-    // Prints 'my.dev.example.org'
+    // Nagpapakita ng 'my.dev.example.org'
     echo $request->host();
 
-Reading the HTTP Method
------------------------
+Pagbabasa ng HTTP na Pamaraan
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Pamamaraan`

File: tl/controllers/request-response.rst [hunk 3]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
+    // Output ng POST
     echo $request->getMethod();
 
-    // Prior to 3.4.0
+    // Bago ang 3.4.0
     echo $request->method();
 
-Restricting Which HTTP method an Action Accepts
------------------------------------------------
+Paghihigpit na kung saan ang HTTP na pamaraan ay isang Aksyon na Tinatanggap
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Pinaghihigpitan kung Anong HTTP na pamamaraan ang Tinatanggap ng Aksyon`

File: tl/controllers/request-response.rst [hunk 4]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
 
-While some apache installs don't make the ``Authorization`` header accessible,
-CakePHP will make it available through apache specific methods as required.
+Habang ang ilang apache na naka-install ay hindi makagawa ng ``Authorization`` na header na mapupuntahan,
+Ang CakePHP ay gagawin itong magagamit sa pamamagitan ng apache na tiyak na mga pamamaraan bilang kinakailangan.
 
 .. php:method:: referer($local = false)
 
-Returns the referring address for the request.
+Binabalik ang nagre-refer sa address para sa hiling.
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `nagre-refer na address`

File: tl/controllers/request-response.rst [hunk 5]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
+----------------------------------
 
-If your application is behind a load balancer or running on a cloud service, you
-will often get the load balancer host, port and scheme in your requests. Often
-load balancers will also send ``HTTP-X-Forwarded-*`` headers with the original
-values. The forwarded headers will not be used by CakePHP out of the box. To
-have the request object use these headers set the ``trustProxy`` property to
+Kung ang iyong aplikasyon ay sa likod ng isang load balancer o tumatakbo sa isang cloud na serbisyo, ikaw
+ay madalas na makakuha ng load balancer na host, port at scheme sa iyong mga hiling. Madalas
+ng load ng mga balander ay nagpapadala din ng ``HTTP-X-Forwarded-*`` na mga header na may orihinal
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `ang load na mga balancer`

File: tl/controllers/request-response.rst [hunk 6]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
-Checking Accept Headers
------------------------
+Pagsusuri sa Tinanggap na mga Header
+------------------------------------
 
 .. php:method:: accepts($type = null)
 
-Find out which content types the client accepts, or check whether it accepts a
-particular type of content.
+Malaman kung ano ang nilalaman ng mga uri sa kliyenteng tinatanggap, o suriin kung ito ay tumatanggap ng
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Alamin kung ano`

File: tl/controllers/request-response.rst [hunk 7]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
-Check for a single type::
+Suriin para sa isang solong uri::
 
     $acceptsJson = $this->request->accepts('application/json');
 
 .. php:method:: acceptLanguage($language = null)
 
-Get all the languages accepted by the client,
-or check whether a specific language is accepted.
+Kunin ang lahat ng mga language na tinatanggap sa kliyente,
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: you could transliterate `language` to `lengguwahe`

File: tl/controllers/request-response.rst [hunk 8]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
+o suriin kung ang tiyak na language ay tinatanggap.
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `lengguwahe`

File: tl/controllers/request-response.rst [hunk 9]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
 
-Get the list of accepted languages::
+Kunin ang listahan sa tinatanggap na mga language::
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `lengguwahe`

File: tl/controllers/request-response.rst [hunk 10]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
 
     $acceptsLanguages = $this->request->acceptLanguage();
 
-Check whether a specific language is accepted::
+Suriin kung ang isang tiyak na language ay tinatanggap::
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `lengguwahe`

File: tl/controllers/request-response.rst [hunk 11]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
 
 .. _request-cookies:
 
-Cookies
--------
+Mga Cookie
+----------
 
-Request cookies can be read through a number of methods::
+Hiling na mga cookie ay maaaring mabasa gamit ang isang bilang ng mga pamamaraan::
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Ang hiling na mga cookie`

File: tl/controllers/request-response.rst [hunk 12]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
-:php:class:`Cake\\Http\\Response` is the default response class in CakePHP.
-It encapsulates a number of features and functionality for generating HTTP
-responses in your application. It also assists in testing, as it can be
-mocked/stubbed allowing you to inspect headers that will be sent.
-Like :php:class:`Cake\\Http\\ServerRequest`, :php:class:`Cake\\Http\\Response`
-consolidates a number of methods previously found on :php:class:`Controller`,
-:php:class:`RequestHandlerComponent` and :php:class:`Dispatcher`. The old
-methods are deprecated in favour of using :php:class:`Cake\\Http\\Response`.
+Ang :php:class:`Cake\\Http\\Response` ay isang default na tugon na class sa CakePHP.
+Ito ay nagpapaikut-ikot sa bilang ng mga tampok atfunctionality para bumuo ng HTTP
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: - please change `nagpapaikut-ikot` to `nag-eencapsulate` instead.
- `sa ilang mga tampok at functionality para sa pagbubuo ng HTTP`

File: tl/controllers/request-response.rst [hunk 13]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
+na mga tugon sa iyong aplikasyon. Ito rin ay tumutulong sa pagsusubok, dahil maaari itong
+naka-mock/naka-stubb na nagpapahintulot sa iyo na siyasatin ang mga header na maipapadala.
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `naka-stub`

File: tl/controllers/request-response.rst [hunk 14]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
+Tulad ng :php:class:`Cake\\Http\\ServerRequest`, :php:class:`Cake\\Http\\Response`
+nagsasama ng bilang ng mga pamamaraan na dati ay nakita sa :php:class:`Controller`,
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `na nagsasama ng ilang mga pamamaraan`

File: tl/controllers/request-response.rst [hunk 15]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
+:php:class:`RequestHandlerComponent` at :php:class:`Dispatcher`. Ang lumang
+mga pamamaraan ay hindi na magagamit sa pabot sa pagagamit sa :php:class:`Cake\\Http\\Response`.
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `sa pabor ng paggamit sa`

File: tl/controllers/request-response.rst [hunk 16]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
 
-``Response`` provides an interface to wrap the common response-related
-tasks such as:
+``Response`` ay nagbibigay ng isang interface para balutin ang karaniwang tugon na may kaugnayad
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `na may kaugnayan`

File: tl/controllers/request-response.rst [hunk 17]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
+na mga gawain tulad sa:
 
-* Sending headers for redirects.
-* Sending content type headers.
-* Sending any header.
-* Sending the response body.
+* Pagpapadala ng mga header para sa mga nagre-redirect.
+* Pagpapadala ng nilalaman na uri ng mga header.
+* Pagpapadala sa anumang header.
+* Pagpapadala ng tugon sa katawan.
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `tugon na katawan`

File: tl/controllers/request-response.rst [hunk 18]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
 
-    // Prior to 3.4.0
+    // Bago ang 3.4.0
     $this->response->type('vcf');
 
-Usually, you'll want to map additional content types in your controller's
-:php:meth:`~Controller::beforeFilter()` callback, so you can leverage the
-automatic view switching features of :php:class:`RequestHandlerComponent` if you
-are using it.
+Karaniwan, gusto mo magmapa sa karagdagang nilalaman na mga uri sa iyong controller sa
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `gusto mong`

File: tl/controllers/request-response.rst [hunk 19]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
-Sending Files
--------------
+Pagpapadala ng mga File
+-----------------------
 
 .. php:method:: withFile($path, $options = [])
 
-There are times when you want to send files as responses for your requests.
-You can accomplish that by using :php:meth:`Cake\\Http\\Response::withFile()`::
+May mga oras na gusto mong mapadala ng mga file bilang mga tugon para sa iyong mga hiling.
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `May mga panahon`

File: tl/controllers/request-response.rst [hunk 20]
```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
...
-:php:meth:`Cake\\Http\\Response::withType()` method.
+Tulad ng ipinakita sa itaas na halimbawa, kailangan mong magpasa sa file na path sa pamamaraan.
+Ang CakePHP ay nagpapadala ng isang tamang nilalaman na uri ng header kung ito ay kilala na file na uri na naka-lista
+sa `Cake\\Http\\Reponse::$_mimeTypes`. Maaari kang magdagdag ng bagong mga uri bago sa pagtawag sa
+:php:meth:`Cake\\Http\\Response::withFile()` sa pamamagitan sa paggamit ng
+:php:meth:`Cake\\Http\\Response::withType()` na pamamaraan.
 
-If you want, you can also force a file to be downloaded instead of displayed in
-the browser by specifying the options::
+Kung gusto mo, maaari ka ding pilitin ang isang file na ma-download sa halip na ipinapakita sa
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `maaari mo ring`

File: tl/controllers/request-response.rst [hunk 21]
```diff
@@ -573,96 +573,96 @@ ics generated on the fly from a string::
...
+Pagtatakda ng mga Header
+------------------------
 
 .. php:method:: withHeader($header, $value)
 
-Setting headers is done with the :php:meth:`Cake\\Http\\Response::withHeader()`
-method. Like all of the PSR-7 interface methods, this method returns a *new*
-instance with the new header::
+Pagtatakda ng mga header ay nagawa na may :php:meth:`Cake\\Http\\Response::withHeader()`
+na pamamaraan. Tulad ng lahat sa PSR-7 na interface na mga pamamaraan, itong pamaraan ay bumabalik ng *new*
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `*bagong*`

File: tl/controllers/request-response.rst [hunk 22]
```diff
@@ -573,96 +573,96 @@ ics generated on the fly from a string::
...
+    // Ilagay ang isang halaga para sa isang umiiral na header
     $response = $response->withAddedHeader('Set-Cookie', 'remember_me=1');
 
-    // Prior to 3.4.0 - Set a header
+    // Bago ang 3.4.0 - Itakda ang isang header
     $this->response->header('Location', 'http://example.com');
 
-Headers are not sent when set. Instead, they are held until the response is
-emitted by ``Cake\Http\Server``.
+Mga headers ay hindi pinadala kung naitakda. Sa halip, sila ay gaganapin hanggang ang tugon ay
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Ang mga header`

File: tl/controllers/request-response.rst [hunk 23]
```diff
@@ -573,96 +573,96 @@ ics generated on the fly from a string::
...
+napalabas sa pamamagitan ng ``Cake\Http\Server``.
 
-You can now use the convenience method
-:php:meth:`Cake\\Http\\Response::withLocation()` to directly set or get the
-redirect location header.
+Maaari mo na ngayong gumamit ng kaginhawaan na pamamaraan sa
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Maaari ka na`

File: tl/controllers/request-response.rst [hunk 24]
```diff
@@ -573,96 +573,96 @@ ics generated on the fly from a string::
...
     $response = $response->withBody($stream);
 
-    // Prior to 3.4.0 - Set the body
+    // Bago ang 3.4.0 - Itakda ang Katawan
     $this->response->body('My Body');
 
-Be sure that ``$stream`` is a :php:class:`Psr\\Http\\Message\\StreamInterface` object.
-See below on how to create a new stream.
+Siguraduhin na ang ``$stream`` ay isang :php:class:`Psr\\Http\\Message\\StreamInterface` na object.
+Tingmam sa ibaba kung papaano lumikha ng isang bagong stream.
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Tingnan`

File: tl/controllers/request-response.rst [hunk 25]
```diff
@@ -573,96 +573,96 @@ ics generated on the fly from a string::
...
 
-You can also stream responses from files using :php:class:`Zend\\Diactoros\\Stream` streams::
+Maaari mo ring mag-stream ng mga tugon mula sa mga file na gamit ang :php:class:`Zend\\Diactoros\\Stream` na mga stream::
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Maaari ka ring`

File: tl/controllers/request-response.rst [hunk 26]
```diff
@@ -671,65 +671,65 @@ stream to the client::
...
-    Disabling caching from SSL domains while trying to send
-    files to Internet Explorer can result in errors.
+    Ang hindi pagpagana sa pag-cache mula sa SSL na mga domain habang sinusubukang ipadala
+    ang mga file sa Internet Explorer ay maaaring magresulta ng mga pagkakamali.
 
 .. php:method:: withCache($since, $time = '+1 day')
 
-You can also tell clients that you want them to cache responses. By using
-:php:meth:`Cake\\Http\\Response::withCache()`::
+Maaari mo ring sabihin ang mga kliyente na gusto mo sa kanila upang i-cache ang mga tugon. Sa pamamagitan sa paggamit
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Maaari mo ring sabihan ang mga kliyente na gusto mong mag-cache sila ng mga tugon.`

File: tl/controllers/request-response.rst [hunk 27]
```diff
@@ -671,65 +671,65 @@ stream to the client::
...
 
-The above would tell clients to cache the resulting response for 5 days,
-hopefully speeding up your visitors' experience.
-The ``withCache()`` method sets the ``Last-Modified`` value to the first
-argument. ``Expires`` header and the ``max-age`` directive are set based on the
-second parameter. Cache-Control's ``public`` directive is set as well.
+Ang sabi sa itaas sa mga kliyente upang i-cache ang resultang tugon hanggang sa limang araw,
+sana ay mapadali ang karanasan ng iyong mga binisita.
+Ang ``withCache()`` na pamamaraan ay nagtatakda ng ``Last-Modified`` na halaga para sa unang
+argumento. ``Expires`` na header at ang ``max-age`` na direktiba ay nakabatay batay sa
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: - `Ang ``Expires`` `
- `direktibo ay nakatakda batay sa`

File: tl/controllers/request-response.rst [hunk 28]
```diff
@@ -671,65 +671,65 @@ stream to the client::
...
+pangalawang parameter. Ang pag-cache na pagkokontrol ng ``public`` direktiba ay itinakda rin.
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `direktibo`

**Review by josteinfrancois — COMMENTED** [review]
Hey @robin-ho. I have reviewed your work. Please see the comments. Thanks.

File: tl/controllers/request-response.rst (hunk 1 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 2 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 3 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 4 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 5 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 6 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 7 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 8 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 9 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 10 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 11 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 12 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 13 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 14 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 15 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 16 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 17 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 18 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 19 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 20 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 21 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 22 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 23 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 24 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 25 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 26 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 27 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 28 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

**Comment by mind-blast** [comment]
I have changed all suggested words. Thank you for the proof read.

**Pull request merged**